### ISSUE TITLE
Add 'label in name' warning to ARIA8

### ISSUE THREAD

**Issue by alastc** [w3c/wcag#667]
From a recent conversation it seems _very_ easy for people to pick up [ARIA8](https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA8) and then fail Label in Name.

Suggest adding a note just above the example to the effect of:

> Note: The success criteria Label in Name requires that the visible label is included as part of the 'accessible name', which is generally set by the aria-label. The examples below meet this requirement.

Also, let's remove the square brackets around the "[Read more", not to open that can of worms!

**Comment by detlevhfischer** [comment]
+1.  Maybe add 2.5.3 before the SC name and put the latter into quotes (or better even, link there).

**Comment by jake-abma** [comment]
+1

**Comment by mbgower** [comment]
+1 I previously tried to scrutinize all the cross-refereneced aria techniques and I saw that this example properly handled the visible text as part of the name. But I agree it would be a good idea to specifically call out the Label in Name consideration.

**Comment by awkawk** [comment]
Let's get a pull request for this and put it on the 4/9 survey.

**Comment by detlevhfischer** [comment]
Possibly, change "requires that the visible label is included" to "requires that text in the visible label is included" to cover both text labels and those that contain text and other stuff (like logos), but exempt purely visual labels (things like the loup icon).
